recipes/sdl_mixer/cmake/conanfile.py

```diff
@@ -0,0 +1,255 @@
+            self.requires("ogg/1.3.5")
+        if self.options.wavpack:
+            # TODO: not available on CCI
+            self.requires("wavpack/x.y.z")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.16 <4]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        rmdir(self, os.path.join(self.source_folder, "external"))
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["BUILD_SHARED_LIBS"] = self.options.shared
+        tc.cache_variables["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
+        if Version(self.version) <= "2.6.3":
+            tc.cache_variables["SDL2MIXER_DEBUG_POSTFIX"] = ""
+        tc.variables["SDL2MIXER_VENDORED"] = False
+        tc.variables["SDL2MIXER_SAMPLES"] = False
+        tc.variables["SDL2MIXER_CMD"] = self.options.cmd
+        if self.version == "2.8.0":
+            # The debug postfix will be removed in an upcoming release
+            tc.variables["SDL2MIXER_DEBUG_POSTFIX"] = ""
+        # WAVE
+        tc.variables["SDL2MIXER_WAVE"] = self.options.wav
+        # FLAC
+        tc.variables["SDL2MIXER_FLAC"] = self.options.flac
+        # GME
+        tc.variables["SDL2MIXER_GME"] = self.options.gme
+        # MOD
+        tc.variables["SDL2MIXER_MOD"] = self.options.modplug or bool(self.options.xmp)
+        tc.variables["SDL2MIXER_MOD_MODPLUG"] = self.options.modplug
+        tc.variables["SDL2MIXER_MOD_XMP"] = self.options.xmp == "libxmp"
+        tc.variables["SDL2MIXER_MOD_XMP_LITE"] = self.options.xmp == "libxmp-lite"
+        # MP3
+        tc.variables["SDL2MIXER_MP3"] = self.options.mpg123 or self.options.minimp3
+        tc.variables["SDL2MIXER_MP3_MPG123"] = self.options.mpg123
+        tc.variables["SDL2MIXER_MP3_MINIMP3"] = self.options.minimp3
+        # MIDI
+        tc.variables["SDL2MIXER_MIDI"] = self.options.get_safe("nativemidi", False) or self.options.get_safe("tinymidi", False) or self.options.fluidsynth
+        tc.variables["SDL2MIXER_MIDI_FLUIDSYNTH"] = self.options.fluidsynth
+        tc.variables["SDL2MIXER_MIDI_TIMIDITY"] = self.options.get_safe("tinymidi", False)
+        tc.variables["SDL2MIXER_MIDI_NATIVE"] = self.options.get_safe("nativemidi", False)
+        # OPUS
+        tc.variables["SDL2MIXER_OPUS"] = self.options.opus
+        # VORBIS
+        if self.options.vorbis == "stb":
+            tc.variables["SDL2MIXER_VORBIS"] = "STB"
+        elif self.options.vorbis == "vorbisfile":
+            tc.variables["SDL2MIXER_VORBIS"] = "VORBISFILE"
+        elif self.options.vorbis == "tremor":
+            tc.variables["SDL2MIXER_VORBIS"] = "TREMOR"
+        else:
+            tc.variables["SDL2MIXER_VORBIS"] = False
+        # WavPack
+        tc.variables["SDL2MIXER_WAVPACK"] = self.options.wavpack
+
+        # TODO: add support for dynamic loading of dependencies
```

Comment:
Was this broken in previous recipe revisions? Seemingly it got supported 🤔

Reply:
No idea. I'm not going to add it in this PR either way.